src/cloud-api-adaptor/docs/initdata.md

```diff
@@ -175,5 +174,24 @@ allow if {
 }
 ```
 
+## Global initdata
```

Comment:
nit: I would drop the `GLOBAL_` prefix from variables. it should be obvious that it's per CAA installation

Reply:
Yes, INITDATA makes sense to me, will make the change.